src/ECL-1.0.xml

```diff
@@ -0,0 +1,44 @@
+<SPDX name="Educational Community License v1.0" identifier="ECL-1.0" osi-approved="true">
+  <urls>
+    <url>http://opensource.org/licenses/ECL-1.0</url>
+  </urls>
+  <header> Copyright (c) 
+    <alt name="copyright" match=".+">&lt;year&gt; &lt;copyright holders&gt;</alt> Licensed under the Educational
+    Community License version 1.0 
+  </header>
+  <license>
+    <title>
+      <p>The Educational Community License</p>
+    </title>
+      <p>This Educational Community License (the "License") applies to any original work of authorship (the
+         "Original Work") whose owner (the "Licensor") has placed the following notice immediately following
+         the copyright notice for the Original Work:</p>
+      <p>Copyright (c) 
+        <alt name="copyright" match=".+">&lt;year&gt; &lt;copyright holders&gt;</alt>
```

Comment:
does this need markup here?  doesn't seem like it'd change in license, but only in standard header